<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clear_data</class>
 <widget class="QDialog" name="clear_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select the appropriate:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="clear_all">
      <property name="text">
       <string>ALL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="clear_professors">
      <property name="text">
       <string>PROFESSORS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="clear_students">
      <property name="text">
       <string>STUDENTS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="clear_classes">
      <property name="text">
       <string>CLASSES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
